Oop tutorial

RomiP opened the pull request:
Added materials for OOP tutorial

RomiP committed:
add OOP tutorial material

RomiP committed:
add index file

RomiP committed:
Update _toc.yml to include OOP tut

scottyhq commented on book/_toc.yml:
```diff
@@ -34,6 +34,12 @@ parts:
         - file: tutorials/sea_ice/1_sea_ice_tutorial
           sections:
             - file: tutorials/sea_ice/2_ATL10_Sentinel-2
+    - file: tutorials/OOP/index
+          title: Object Oriented Programming
+          sections:
+            - file: tutorials/OOP/OOP_slides
+            - file: tutorials/OOP/OOP_tutorial_complete
+            - file: tutorials/OOP/alien
```
```suggestion
        - file: tutorials/sea_ice/1_sea_ice_tutorial
          sections:
            - file: tutorials/sea_ice/2_ATL10_Sentinel-2
        - file: tutorials/OOP/index
          title: Object Oriented Programming
          sections:
            - file: tutorials/OOP/OOP_slides
            - file: tutorials/OOP/OOP_tutorial_complete
            - file: tutorials/OOP/alien
```

(match indent)

jomey committed:
TOC - Fix indent

Co-authored-by: Scott Henderson <scottyhq@gmail.com>

jomey committed:
Tutorial - OOP - Change link to tutorial PDF slides

PDFs can not be linked directly with JupyterBook and need a full URL to
open in a browser.

jomey commented:
@RomiP - JupyterBook can not link directly to PDF documents unfortunately and it requires them to be linked per URL. I changed the links for your tutorial to have the slide link on the [index](https://deploy-preview-63--icesat-2-2023.netlify.app/tutorials/oop/) of that section.
Will that work for you?
